src/main/java/com/commercetools/sync/sdk2/customobjects/helpers/CustomObjectCompositeIdentifier.java

```diff
@@ -0,0 +1,139 @@
+package com.commercetools.sync.sdk2.customobjects.helpers;
+
+import static java.lang.String.format;
+
+import com.commercetools.api.models.custom_object.CustomObject;
+import com.commercetools.api.models.custom_object.CustomObjectDraftBuilder;
+import java.util.Objects;
+import javax.annotation.Nonnull;
+
+/**
+ * This class is only meant for the internal use of the commercetools-sync-java library.
+ *
+ * <p>Since there is no one unique identifier for customObjects on the commercetools API, we
+ * uniquely identify a customObject as a composite id of the following fields: key, container. This
+ * class models such composite key.
+ */
+public final class CustomObjectCompositeIdentifier {
+
+  private final String key;
+  private final String container;
+  private static final String WRONG_FORMAT_ERROR_MESSAGE =
+      "The custom object identifier value: \"%s\" does not have "
+          + "the correct format. The correct format must have a vertical bar \"|\" character "
+          + "between the container and key.";
+
+  private CustomObjectCompositeIdentifier(
+      @Nonnull final String key, @Nonnull final String container) {
+    this.key = key;
+    this.container = container;
+  }
+
+  /**
+   * Given a {@link io.sphere.sdk.customobjects.CustomObjectDraft}, creates a {@link
```

Comment:
```suggestion
   * Given a {@link  com.commercetools.api.models.custom_object.CustomObjectDraft}, creates a {@link
```

Reply:
Please check and replace in all comments.

Reply:
https://github.com/commercetools/commercetools-sync-java/pull/928/commits/a1c4729eff2306e52b861bc81501348bbbe42a54